src/components/cl/hier/cl_hier_context.c

```diff
@@ -23,6 +24,13 @@ UCC_CLASS_INIT_FUNC(ucc_cl_hier_context_t,
 
     UCC_CLASS_CALL_SUPER_INIT(ucc_cl_context_t, cl_config,
                               params->context);
+    if (params->params.oob.n_oob_eps == 1) {
```

Comment:
oob field is not always available in context params, need to check if  (params->params & UCC_CONTEXT_PARAM_FIELD_OOB) == 1